internal/aks/client.go

```diff
@@ -94,11 +94,15 @@ func GetSecrets(secretsCache wranglerv1.SecretCache, spec *aksv1.AKSClusterConfi
 		return nil, fmt.Errorf("couldn't find secret [%s] in namespace [%s]", id, ns)
 	}
 
+	tenantIDBytes := secret.Data["azurecredentialConfig-tenantId"]
```

Comment:
You may want to add this to the example in the README where secret creation is demonstrated.

Reply:
I have opened new PR https://github.com/rancher/aks-operator/pull/15